zebra-consensus/src/error.rs

```diff
@@ -22,8 +22,11 @@ pub enum SubsidyError {
     #[error("founders reward output not found")]
     FoundersRewardNotFound,
 
-    #[error("funding stream output not found")]
-    FundingStreamNotFound,
+    #[error("funding stream output with value not found")]
+    FundingStreamValueNotFound,
+
+    #[error("funding stream output with address not found")]
```

Comment:
Nit: I'm not sure if my suggestion is correct, but would it be clearer to remove the `with` in both cases?

```suggestion
    #[error("funding stream output value not found")]
    FundingStreamValueNotFound,

    #[error("funding stream output address not found")]
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/3040/commits/0915d8bfd42d873961d8e0a2d5cf42c064b5b1ea